client/src/components/BannerSection.tsx

```diff
@@ -44,15 +45,23 @@ function BannerText() {
 
 function BannerSection() {
   const { classes } = useStyles();
-  const history = useHistory();
+  const modalContext = useContext(ModalContext);
+  const { openModal } = modalContext;
```

Comment:
I think this will also work for this
```suggestion
  const { openModal } = useContext(ModalContext);
```